app/views/pages/home.html.erb

```diff
@@ -1,158 +1,160 @@
+    </section>
+
+    <section class="my-story container-1000">
+        <div class="my-story__left">
+            <a href='/life-as-a-teacher/real-stories/' aria-label="Life as a teacher">
+                <div class="my-story__left__video" style="background-image: url('<%= asset_pack_path 'media/images/paulsstory.png' %>')">
+                    <div class="my-story__left__video__overlay">
+                        <div class="icon-play"></div>
+                    </div>
                 </div>
             </a>
         </div>
-    </div>
-</section>
-
-<section class="my-story container-1000">
-    <div class="my-story__left">
-        <a href='/life-as-a-teacher/real-stories/' aria-label="Life as a teacher">
-            <div class="my-story__left__video" style="background-image: url('<%= asset_pack_path 'media/images/paulsstory.png' %>')">
-                <div class="my-story__left__video__overlay">
-                    <div class="icon-play"></div>
-                </div>
+        <div class="my-story__right">
+            <div class="my-story__right__header">
+                <h1 class="strapline-image">My story into teaching</h1>
             </div>
-        </a>
-    </div>
-    <div class="my-story__right">
-        <div class="my-story__right__header">
-            <h1 class="strapline-image">My story into teaching</h1>
+            <p>As a newly qualified teacher, Paul was keen to move up the teaching career ladder</p>
+            <p>
+                <a class="git-link" href='/life-as-a-teacher/real-stories/'>
+                    Read Paul's story and others <i class="fas fa-chevron-right"></i>
+                </a>
+            </p>
         </div>
-        <p>As a newly qualified teacher, Paul was keen to move up the teaching career ladder</p>
-        <p>
-            <a class="git-link" href='/life-as-a-teacher/real-stories/'>
-                Read Paul's story and others <i class="fas fa-chevron-right"></i>
-            </a>
-        </p>
-    </div>
-</section>
+    </section>
 
-<section class="pic-links">
-    <div class="pic-links__pic" style="background-image: url('<%= asset_pack_path 'media/images/piclink1.png' %>')">
-        <a href="/life-as-a-teacher/index">
-            <div class="call-to-action-button">Teaching as a career  <i class="fas fa-chevron-right"></i></div>
-        </a>
-    </div>
-
-    <div class="pic-links__pic" style="background-image: url('<%= asset_pack_path 'media/images/piclink2.png' %>')">
-        <div class="pic-links__pic__mask"></div>
-        <div class="pic-links__pic__salaries">
-            <a href="/life-as-a-teacher/teachers-salaries-and-benefits/">
-                <div class="call-to-action-button">Salaries and benefits  <i class="fas fa-chevron-right"></i></div>
+    <section class="pic-links">
+        <div class="pic-links__pic" style="background-image: url('<%= asset_pack_path 'media/images/piclink1.png' %>')">
+            <a href="/life-as-a-teacher/index">
```

Comment:
Unrelated to this change, but this URL looks odd -- why isn't it just `/life-as-a-teacher/`?

Reply:
not sure, the content driven part was in place before I joined, its worth flagging